_data/navigation.yml

```diff
@@ -165,6 +165,10 @@ setup:
             url: /setup/productionize/caching/configure-redis-usage/
           - title: Externalize Redis
             url: /setup/productionize/caching/externalize-redis/
+      - title: Configure Scaling
+        collapsed: true
+          - title: Horizontally Scale Spinnaker
+            url: /setup/productionize/scaling/horizontal-scaling/
```

Comment:
Will there be a separate article on vertical scaling?

Reply:
Yes